wurst/data/LinkedList.wurst

```diff
@@ -516,5 +516,46 @@ public function group.asList() returns LinkedList<unit>
 		result.add(u)
 	return result
 
+public function LinkedList<T>.getLast<T>() returns T
```

Comment:
Why not use the LinkedList's method to get the last element which is O(1)?
The `.get` call is O(n)

Reply:
I just moved this function from StringUtils, mostly to just keep dependencies from failing... I did not create this function, it was there for some time already :-).

Reply:
the getals was not existing i think once we created it ^^

Reply:
You shouldn't mark stuff as resolved which isn't. I still see no point in this - either present a reason or remove.

Reply:
Removed.

Reply:
well Linkedlist only has:
.peek()
and 
LinkedList.backiterator().next()
while peek does not truely provide the same functionality

Reply:
`getFirst` and `peek` return last/first added elements